src/shared/components/title-fades/title-fades.component.scss

```diff
@@ -1,96 +1,89 @@
+  position: relative;
+  overflow: hidden;
+  height: 32px;
+  background: linear-gradient(270deg, transparent, rgba(255, 116, 24, 0.8) 20%);
+  h2 {
+    padding-top: 7px;
     height: 32px;
-    background: linear-gradient(270deg, transparent , #FF7418 20%);
-    h2{
-        padding-top: 7px;
-        height: 32px;
-        font-weight: 700;
-        font-size: 18px;
-        padding-left: 80px;
-        line-height: 21.6px; 
-        color: #0D0D0D;
-        position: relative;
-
-     }
-     .first-fade{
-        border: 9px solid black;
-        height: 200px;
-        width: 200px;
-        position: absolute;
-        transform: translateY(-50%) rotate(45deg);
-        top: 50%;
-        right: 92%;
-     }
-
-     .second-fade{
-        border: 9px solid black;
-        height: 200px;
-        width: 200px;
-        position: absolute;
-        transform: translateY(-50%) rotate(45deg);
-        top: 50%;
-        right: 98%;
-     }
+    font-weight: 700;
+    font-size: 18px;
+    padding-left: 80px;
+    line-height: 21.6px;
+    color: #0d0d0d;
+    position: relative;
+  }
+  .first-fade {
+    border: 9px solid black;
+    height: 200px;
+    width: 200px;
+    position: absolute;
+    transform: translateY(-50%) rotate(45deg);
+    top: 50%;
+    right: 92%;
+  }
 
+  .second-fade {
+    border: 9px solid black;
+    height: 200px;
+    width: 200px;
+    position: absolute;
+    transform: translateY(-50%) rotate(45deg);
+    top: 50%;
+    right: 98%;
+  }
 }
 
-
-
 // If needed change the background
-.upcoming-events-title-yellow{
-   position: relative;
-   overflow: hidden;
-   height: 32px;
-   background: linear-gradient(270deg, transparent , #FFDC23 20%);
-   h2{
-       padding-top: 7px;
-       height: 32px;
-       font-weight: 700;
-       font-size: 18px;
-       padding-left: 80px;
-       line-height: 21.6px; 
-       color: #0D0D0D;
-       position: relative;
-
-    }
-    .first-fade{
-       border: 9px solid black;
-       height: 200px;
-       width: 200px;
-       position: absolute;
-       transform: translateY(-50%) rotate(45deg);
-       top: 50%;
-       right: 92%;
-    }
-
-    .second-fade{
-       border: 9px solid black;
-       height: 200px;
-       width: 200px;
-       position: absolute;
-       transform: translateY(-50%) rotate(45deg);
-       top: 50%;
-       right: 98%;
-    }
+.upcoming-events-title-yellow {
+  position: relative;
+  overflow: hidden;
+  height: 32px;
+  // fix gradients
+  background: linear-gradient(270deg, transparent, rgba(255, 220, 35, 0.8) 20%);
+  h2 {
+    padding-top: 7px;
+    height: 32px;
+    font-weight: 700;
+    font-size: 18px;
+    padding-left: 80px;
+    line-height: 21.6px;
+    color: #0d0d0d;
+    position: relative;
+  }
+  .first-fade {
+    border: 9px solid black;
+    height: 200px;
+    width: 200px;
+    position: absolute;
+    transform: translateY(-50%) rotate(45deg);
+    top: 50%;
+    right: 92%;
+  }
 
+  .second-fade {
```

Comment:
Aren't the 90% of the styles of each "fade" (.first-fade , .second-fade) the same?

Reply:
Yes, thats good coment. Fixed.